wire-ios/Wire-iOS/Sources/Helpers/UIApplication/UIApplication+Permissions.swift

```diff
@@ -67,11 +67,13 @@ extension UIApplication {
                     grantedHandler(false)
                 case .authorized:
                     grantedHandler(true)
+                case .limited:
```

Comment:
what's the diff with restricted?

Reply:
`.restricted` means, the user can't give access (e.g. due to parental controls) while `.limited` means the user decided that the app gets only limited access.

Reply:
ok